src/pages/HomePage/HomePage.tsx

```diff
@@ -1,23 +1,20 @@
 import React from "react";
-import GoogleLogin, {
-  GoogleLoginResponse,
-  GoogleLoginResponseOffline,
-} from "react-google-login";
 import PageTemplate from "../TemplatePage/PageTemplate";
 import { useNavigate } from "react-router-dom";
-import { loginService } from "../../services/application/loginService";
 import logo from "../../images/teaminator_logo.png";
-import { Settings } from "../../services/infrastructure/Settings";
 import "./homepage.css";
+import LoginButton from "../../components/LoginButton/LoginButton";
+import { PageRoutes } from "../pageRoutes";
 
 function HomePage() {
   const navigate = useNavigate();
 
-  const googleLoginHandler = async (
-    res: GoogleLoginResponse | GoogleLoginResponseOffline
-  ) => {
-    const route = await loginService(res);
-    navigate(route);
+  const redirectTo = (isSuccessful: boolean) => {
+    if (isSuccessful) {
+      navigate(PageRoutes.TEAMS);
+      return;
```

Comment:
is better have return than a else?

Reply:
For me it is,  I can change it if you don't agree

Reply:
I don't mind keeping it like this.

Reply:
Maybe we can start to apply this all around the code so we follow the same rules

Reply:
👍